dash/include/dash/internal/Logging.h

```diff
@@ -20,10 +20,6 @@
 #include <sys/types.h>
 #include <unistd.h>
 
-namespace dash {
-  // forward-declaration
-  global_unit_t myid();
-}
 
```

Comment:
That cannot be removed, as logging is used in Team.cc and vise versa.
Error (gcc5):
```bash
In file included from /opt/dash/dash/include/dash/util/Timer.h:20:0,                                  │ 1 file changed, 4 insertions(+)
                 from /opt/dash/dash/src/util/TimestampClockPosix.cc:1:                               │ felix@HIGHPOWER  ~/repos/github-dash/dash   feat-hdf5-views ●  git commit -a
/opt/dash/dash/include/dash/internal/Logging.h: In function 'void dash::internal::logging::Log_Line(co│[feat-hdf5-views aaa90a2] skip ASIO test if using mpich, as ASIO unrelated concurrency problems occure
nst char*, const char*, int, const char*, const string&)':                                            │ 1 file changed, 10 insertions(+), 2 deletions(-)
/opt/dash/dash/include/dash/internal/Logging.h:194:29: error: 'myid' is not a member of 'dash'        │ felix@HIGHPOWER  ~/repos/github-dash/dash   feat-hdf5-views  git push
   dash::global_unit_t uid = dash::myid();
```